lib/iris/tests/test_cell.py

```diff
@@ -130,15 +130,15 @@ class Terry:
         self.assertEqual(self.d.__ne__(Terry()), NotImplemented)
 
     def test_numpy_int_equality(self):
-        dtypes = (np.int, np.int16, np.int32, np.int64)
+        dtypes = (int, np.int16, np.int32, np.int64)
```

Comment:
Given that this method name contains 'numpy' I'm not sure if a bare `int` should remain here now.

Reply:
```suggestion
        dtypes = (np.int_, np.int16, np.int32, np.int64)
```

Reply:
@akuhnregnier Great spot and attention to detail :eyes: 

I think honouring the intent of the test, we could opt for `np.int_` :+1: